### PR TITLE
Feature/controller

### PR DESCRIPTION
**Pull request by Aqurds**
The following features were added,

**Added RSpec as a gem to your project.**
**Created a Request spec file for all your controllers.**
**Make sure that for each action you check:**
- > If the response status was correct.
- > If a correct template was rendered.
- > If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/test_controllers/test_post_controller.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+    it 'renders "show template" with right text' do
+      expect(response.body).to match('Post test')
+      expect(response.body).to match('Show Comments')
+    end
+
+    it 'does not render a different template' do
+      expect(response).to_not render_template(:index)
+    end
+  end
```
Comment (DeliceLydia) [inline spec/test_controllers/test_post_controller.rb]: - [x] I am not able to run `spec` I suggest that you change the name of the folder from `test_controllers` to `spec/requests/`
Screenshot 2023-10-04 at 10 33 53

Comment (DeliceLydia) [reply spec/test_controllers/test_post_controller.rb]: - [x] Change names of the files from `test_post_controller.rb` to `posts_spec.rb` and `users_spec.rb`

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Aqurds, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Used RSpec gem for testing
- Used `type: request`
- No linter errors


## Required Changes ♻️
_Check the comments under the review._



### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/test_controllers/test_post_controller.rb (hunk 1 above)
Comment (Aqurds) [reply spec/test_controllers/test_post_controller.rb]: Changes implemented.

**Review by Olamarx — APPROVED** [review]
Hi @Aqurds, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- All test cases passed ✔️
- Commit messages meaningful ✔️
- Gitflow ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**